test/spec/controllers/exportimport.js

```diff
@@ -213,49 +204,4 @@ describe('Controller: ExportImportCtrl', function () {
     expect(scope.downloadLink.indexOf('blob:http://localhost:8080/')).to.be.above(-1);
   });
```

Comment:
There are a number of function attached to the scope that can be tested for  this controller. I think at least the important ones should have test.